lib/gooddata/lcm/lcm2.rb

```diff
@@ -323,7 +330,7 @@ def perform(mode, params = {})
           # Invoke action
           begin
             GoodData.logger.info("Running #{action.name} action ...")
-
+            params.clear_filters
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)